docs/_documentations/vsc-uninstall.md

```diff
@@ -1,20 +1,23 @@
 ---
 layout: docs
 title: Uninstalling Codewind for VS Code
-description: Uninstalling Codewind for VS Code
+description: How to uninstall Codewind from VS Code
```

Comment:
Please could you add a longer description - given that the 1st 2 lines will be exposed in the website search? Also to ensure there is enough description so that it is easily distinguished from other topics, guides, blogs etc. on search.

 e.g. something along the lines of:  'Instructions for how to uninstall Codewind. Uninstalling Codewind consists of 2 steps. First, uninstall the Codewind containers and images. Then proceed to uninstall Codewind'

Reply:
Rectified.